pysilcam/plotting.py

```diff
@@ -259,12 +261,24 @@ def summarise_fancy_stats(stats_file, config_file, monitor=False,
 
     while True:
         try:
-            montage = sc_pp.make_montage(stats_file,
-                                         settings.PostProcess.pix_size,
-                                         roidir=settings.ExportParticles.outputpath,
-                                         auto_scaler=msize * 2, msize=msize,
-                                         maxlength=maxlength,
-                                         oilgas=oilgas)
+            if crop_stats:
+                montage = sc_pp.make_montage(
+                    stats_file,
+                    settings.PostProcess.pix_size,
+                    roidir=settings.ExportParticles.outputpath,
+                    auto_scaler=msize * 2, msize=msize,
+                    maxlength=maxlength,
+                    oilgas=oilgas,
+                    crop_stats_bounds=settings.PostProcess.img_crop)
+            else:
+                montage = sc_pp.make_montage(
+                    stats_file,
+                    settings.PostProcess.pix_size,
+                    roidir=settings.ExportParticles.outputpath,
+                    auto_scaler=msize * 2, msize=msize,
+                    maxlength=maxlength,
+                    oilgas=oilgas,
+                    crop_stats_bounds=None)
```

Comment:
Instead of duplicating the function call inside another nesting level (if/else), I suggest:
montage = sc_pp.make_montange(...., crop_stats_bounds=getattr(settings.PostProcess, 'img_crop', None)

Reply:
I would like a solution like this, as its much more elegant. But that's slightly different functionality, as if crop_stats is False, this could still give a cropped rectangle. So I want exactly your solution above, but that's only if crop_stats is True, and then "None" if it's false.

I couldn't think how else to do this other than an if/else statement around the function call.

Reply:
Am testing:
`....  crop_stats_bounds=settings.PostProcess.img_crop if crop_stats else None)`

Reply:
Just to note in case we look back at this: We took a different approach now, and there is no crop_stats_bounds variable (only crop stats, which is None or a 4-vector).